python/ray/autoscaler/local/node_provider.py

```diff
@@ -90,9 +92,12 @@ class LocalNodeProvider(NodeProvider):
 
     def __init__(self, provider_config, cluster_name):
         NodeProvider.__init__(self, provider_config, cluster_name)
-        self.state = ClusterState("/tmp/cluster-{}.lock".format(cluster_name),
-                                  "/tmp/cluster-{}.state".format(cluster_name),
-                                  provider_config)
+        self.state = ClusterState(
```

Comment:
Let's not touch the autoscaler in this PR. It is only used for running ray on a cluster, right now the focus should be on getting the local version of ray running on windows. The autoscaler is really hard to test, so we should not change it.

Reply:
Sure, I reverted the autoscaler changes.